src/components/navbar/Navbar.css

```diff
@@ -0,0 +1,44 @@
+.navbar {
+  width: 100%;
+  height: 5.5rem;
+  background-color: var(--backgroundcolor);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.logo h3 {
+  font-size: 2.5rem;
+  margin-left: 2rem;
+  color: var(--primarycolor);
+}
+.search_bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.search_bar input {
+  width: 27rem;
```

Comment:
you can also use percentage to provide width with respect to your header container

Reply:
Thanks for the suggeston @dev-prabhat , I will use %